media.css

```diff
@@ -0,0 +1,1400 @@
+    justify-content: center;
+    order: 1;
+  }
+
+  .video {
+    display: none;
+  }
+
+  .video-adaptive {
+    display: flex;
+    justify-content: center;
+    width: 74%;
+    margin: 0 auto;
+  }
+
+  .video-section__content {
+    display: flex;
+    flex-direction: column;
+    order: -1;
+    width: 74%;
+    align-items: center;
+  }
+
+  .video-content__h3 {
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 15px;
+    color: #fff;
+    text-transform: uppercase;
+    width: 100%;
+    text-align: center;
+  }
+
+  .video-content__description {
+    width: 100%;
+    font-size: 14px;
+    line-height: 17px;
+    margin-bottom: 41px;
+    color: #fff;
+    text-align: center;
+  }
+
+  .stories-content {
+    width: 100%;
+    padding-bottom: 105px;
+    background-color: #000;
+    color: #fff;
+    margin-top: -5px;
+  }
+
+  .stories-content__h4 {
+    font-size: 20px;
+    line-height: 19px;
+    font-weight: bold;
+    text-align: center;
+    text-transform: uppercase;
+    margin-bottom: 25px;
+    color: #fff;
+  }
+
+  .stories-items {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .stories-items__item {
+    width: 75%;
+  }
+
+  .stories-item__image {
+    display: block;
+    width: 100%;
+    margin: 0 auto;
+    margin-bottom: 29px;
+  }
+
+  .stories-item__title {
+    font-size: 20px;
+    line-height: 19px;
+    font-weight: bold;
+    margin-bottom: 9px;
+  }
+
+  .stories-item__description {
+    font-size: 14px;
+    line-height: 17px;
+    margin-bottom: 29px;
+  }
+
+  .stories-item__description-extended {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 0.7fr 0.5fr 1fr;
+  }
+
+  .hidden-1024 {
+    display: flex;
+    align-items: center;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+    line-height: 14px;
+    font-weight: bold;
+    color: #d1d1d1;
+    opacity: 1;
+    text-align: right;
+  }
+
+  .gallery {
+    display: block;
+    height: 17%;
+    background: linear-gradient(2.6deg, #fff 50%, #000 51%);
+    margin-bottom: -5px;
+    width: 100%;
+  }
+
+  .gallery__title {
+    font-size: 25px;
+    line-height: 24px;
+    color: #e8e8e8;
+    opacity: 1;
+    text-align: center;
+    text-transform: uppercase;
+    font-weight: bold;
+    margin-bottom: 14px;
+    margin-right: 0;
+  }
+
+  .gallery-content {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 3;
+    margin-left: 48px;
+    margin-right: 48px;
+  }
+
+  .gallery-content__item {
+    display: block;
+    margin-bottom: -4px;
+  }
+
+  .gallery__photo {
+    width: 100%;
+  }
+
+  .hidden-1024 {
+    display: block;
+  }
+
+  .footer {
+    width: 100%;
+  }
+
+  .footer__content {
+    max-width: 75%;
+    padding-top: 38px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .footer__column {
+    text-align: center;
+    margin-bottom: 45px;
+  }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)